File: VB/WindowsFormsApplication202/Form1.vb
Imports DevExpress.Utils
Imports DevExpress.XtraEditors.CustomEditor
Imports DevExpress.XtraGrid.Views.Grid
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace WindowsFormsApplication202
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
            gridControl1.DataSource = GetData(5)
			ImageSliderHelper.BindImageSlider(gridControl1, "Image", imageCollection1)
		End Sub
		Private Function GetData(ByVal rows As Integer) As DataTable
			Dim dt As New DataTable()
			dt.Columns.Add("Description", GetType(String))
			dt.Columns.Add("Image", GetType(Image))
            For i As Integer = 0 To rows - 1
                dt.Rows.Add(i.ToString(), imageCollection1.Images(i))
            Next i
			Return dt
		End Function
	End Class
End Namespace

File: CS/WindowsFormsApplication202/Form1.cs
using DevExpress.Utils;
using DevExpress.XtraEditors.CustomEditor;
using DevExpress.XtraGrid.Views.Grid;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace WindowsFormsApplication202 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            gridControl1.DataSource = GetData(5);
            ImageSliderHelper.BindImageSlider(gridControl1, "Image", imageCollection1);
        }
        DataTable GetData(int rows) {
            DataTable dt = new DataTable();
            dt.Columns.Add("Description", typeof(string));
            dt.Columns.Add("Image", typeof(Image));
            for (int i = 0; i < rows; i++) {
                dt.Rows.Add(i.ToString(), imageCollection1.Images[i]);
            }
            return dt;
        }
    }
}
